plugins/modules/cloud/scaleway/scaleway_container_registry_info.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Scaleway Serverless container registry info module
+#
+# Copyright (c) 2022, Guillaume MARTINEZ <lunik@tiwabbit.fr>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: scaleway_container_registry_info
+short_description: Scaleway Container registry info module
+version_added: 5.8.0
+author: Guillaume MARTINEZ (@Lunik)
+description:
+  - This module return info on container registry on Scaleway account.
```

Comment:
Again a bit of nit picking on my side. Maybe better as:
```suggestion
  - This module return information about a container registry on Scaleway account.
```
?

Reply:
Fixed in ac03fd5c5b30d99f508cb2699f9ff3513fabc44a